scanners/reactnative-expo/reactnative-expo.go

```diff
@@ -67,461 +124,451 @@ func (scanner *Scanner) DetectPlatform(searchDir string) (bool, error) {
 	}
 
 	log.TPrintf("%d package.json file detected", len(packageJSONPths))
+	for _, pth := range packageJSONPths {
+		log.TPrintf("- %s", pth)
+	}
+	log.TPrintf("")
 
-	log.TInfof("Filter relevant package.json files")
+	log.TInfof("Filter package.json files with expo dependency")
 
 	relevantPackageJSONPths := []string{}
-	iosScanner := ios.NewScanner()
-	androidScanner := android.NewScanner()
-
 	for _, packageJSONPth := range packageJSONPths {
-		dependencyFound, err := FindDependencies(packageJSONPth, "expo", "eject")
+		packages, err := cordova.ParsePackagesJSON(packageJSONPth)
 		if err != nil {
-			return false, err
-		}
-
-		// eject or expo script/dependency not found
-		if !dependencyFound {
+			log.Warnf("Failed to parse package json file: %s, skipping...", packageJSONPth)
 			continue
 		}
 
-		if err := ensureNodeModules(filepath.Dir(packageJSONPth)); err != nil {
-			log.Errorf("ERROR DURING INSTALLING NPM: %s", err)
-			return false, nil
+		_, found := packages.Dependencies["expo"]
```

Comment:
will it be found in case if there is a version or anything marked for this dependency?

Reply:
yep as version is the value of the field: https://github.com/bitrise-samples/BitriseCRNA/blob/master/package.json#L12